src/world/Actor/Chara.h

```diff
@@ -183,7 +183,7 @@ namespace Sapphire::Entity
 
     std::string getName() const;
 
-    bool face( const Common::FFXIVARR_POSITION3& p );
+    bool face( const Common::FFXIVARR_POSITION3& p , float epsilon = 0.005f);
```

Comment:
don't do it this way, if you want to check the epsilon use `std::numeric_limits< float >::epsilon` instead of hardcoding it in the prototype